### ISSUE TITLE
A way to exclude files in a compliation

### ISSUE THREAD

**Issue by reduxdj** [brunch/brunch#720]
Whether its CSS or JS files, is there a way to exclude certain folders from a config... for instance, if we are using overrides to essentially as a build target, how can I give a set of files not to be compressed into app.js or app.css?

If this is not a feature - I'd like to +1 this.

**Comment by istr** [comment]
It think that this is a rather common use case. Look at all the templates that just "rename the file to start with an underscore" copying over some upstream repo (e.g. https://github.com/Anaphase/brunch-banana-pancakes just renaming the full bootstrap .less source tree). This is not feasible to keep up with upstream dependencies that have different conventions (there are *lots of them*). The primers and overall documentation should clearly state how to adapt this convention that could only be considered "rather uncommon". The "convention over configuration" approach is good, but kind of pushes this great tool into a niche position it does not deserve, because it does not play well with external upstream dependencies not following its own conventions.

**Comment by es128** [comment]
Answered @istr's question (added to these two super-old issues) at https://github.com/brunch/brunch/issues/189#issuecomment-143470566

tl;dr there's also a `conventions.ignored` setting and globs can be negated for excluding files

**Comment by istr** [comment]
Yes, figured it out, after some hours of re-reading docs and trying to get grips on ```anymatch``` logic (e.g. could not get it to work with globs instead of REs inside of ```ignored``` anymatch set).

A minimal working example somewhere in the docs would be nice, like

```
exports.config = {
  optimize: true,
  modules: {
    addSourceUrls: true
  },
  server: {
    run: false
  },
  paths: {
    public: 'web/',
    watched: [
      'src/',
      'web/uikit/src/',
      'web/uikit/custom/'
    ]
  },
  conventions: {
    ignored: [
      /^web\/uikit/,
      /^src\/less\/block/,
      /^src\/less\/common/,
      /^src\/less\/view/
    ],
  },
  files: {
    javascripts: {
      joinTo: {}
    },
    stylesheets: {
      defaultExtension: 'less',
      joinTo: {
        'css/style-user.css': 'src/less/style-user.less',
        'css/style-anonymous.css': 'src/less/style-anonymous.less',
        'css/style-cordova.css': 'src/less/style-cordova.less'
      }
    }
  }
};
```

I guess that there are scores of "legacy" projects out there using a "single xyz file, requiring / including the whole shebang" that would need to utilize a very similar setup to get brunch running.
